client/signup/step-wrapper/index.jsx

```diff
@@ -202,13 +199,17 @@ class StepWrapper extends Component {
 			'has-navigation': hasNavigation,
 		} );
 
+		let sticky = false;
+		if ( isSticky !== undefined ) {
+			sticky = isSticky;
+		} else {
+			sticky = isReskinnedFlow( flowName ) ? true : false;
```

Comment:
Changing it back to the original value `sticky = isReskinnedFlow( flowName ) ? null : false;` to maintain the sticky bar position at the top in this PR:
https://github.com/Automattic/wp-calypso/pull/82558